tests/test_stations.py

```diff
@@ -1,18 +1,59 @@
 # -*- coding: utf-8 -*-
-
+import os
 import unittest
 
+from openkongqi.conf import config_from_object
 from openkongqi.utils import get_uuid
 
 
+here = os.path.abspath(os.path.dirname(__file__))
+TEST_DATA_PATH = os.path.join(here, 'data', 'stations')
+
+class Conf(object):
+    settings = {
+        'STATIONS_MAP_DIR': TEST_DATA_PATH
+    }
+
+confobj = Conf()
+config_from_object(confobj)
+
+from openkongqi.stations import get_station_map
+
+
 class TestStations(unittest.TestCase):
     """Test the stations UUID map functions"""
 
     def test_get_uuid(self):
-        """test UUID generation"""
+        """Test UUID generation"""
         self.assertEqual(
             get_uuid('cn', 'shanghai', 'jinganjincezhan'),
             'cn:shanghai:jinganjincezhan')
         self.assertEqual(
             get_uuid('cn:shanghai', 'jinganjincezhan'),
             'cn:shanghai:jinganjincezhan')
+
+    def test_get_stations_uuid(self):
+        """Test station map generation"""
+        self.assertEqual(
+            get_station_map('us:oh'),
+            {'Cleveland Station': {'uuid': 'cleveland'}, 'Cincinnati Station': {'uuid': 'cincinnati'}}
```

Comment:
in tests as there's a lot of "visual" validation in case of error, I find it best to make the data structures as easy to read. So here I would write:
```py
{
    'Cleveland Station': {
        'uuid': 'cleveland'
    },
    'Cincinnati Station': {
        'uuid': 'cincinnati'
    }
}
```